Source/WebCore/platform/cocoa/ParentalControlsContentFilter.h

```diff
@@ -57,25 +55,20 @@ class ParentalControlsContentFilter final : public PlatformContentFilter {
 #endif
     
 private:
-#if HAVE(WEBCONTENTRESTRICTIONS)
-    static bool enabled(bool usesWebContentRestrictions);
-#else
-    static bool enabled();
-#endif
+    explicit ParentalControlsContentFilter(const PlatformContentFilter::FilterParameters&);
+    bool enabled() const;
 
-    ParentalControlsContentFilter() = default;
     void updateFilterState();
 #if HAVE(WEBCONTENTRESTRICTIONS)
-    void updateFilterState(bool shouldBlock, NSData *replacmentData);
-    void setUsesWebContentRestrictions(bool) final;
+    void updateFilterState(PlatformContentFilter::State, NSData*);
```

Comment:
Please use `NSData *replacementData`, with the star on the right side and the typo fixed in the argument name (which I think we shouldn't omit).